tests/bud.bats

```diff
@@ -43,6 +43,13 @@ load helpers
   expect_output --substring "unsupported value"
 }
 
+@test "build with inline RUN --network=default" {
+  run_buildah build --network=host $WITH_POLICY_JSON -t source -f $BUDFILES/inline-network/Dockerfile4
+  build1="$output"
```

Comment:
This should be the same as the network namespace that this test itself is running in.

Reply:
Done.